source/retryable-writes/tests/unsupported-multi-statement.yml

```diff
@@ -0,0 +1,88 @@
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+
+minServerVersion: '3.6'
+maxServerVersion: '3.6.99'
```

Comment:
I think we should omit maxServerVersion until we schedule retryable multi-doc operations. Otherwise we'll stop testing this in 3.7+ even though it might still be expected behavior.